iot-hub/Samples/device/ImportExportDevicesSample/ProcessImpExpCommands.cs

```diff
@@ -0,0 +1,438 @@
+﻿using Microsoft.Azure.Devices;
+using Microsoft.Azure.Devices.Common;
+using Microsoft.Azure.Storage;
+using Microsoft.Azure.Storage.Blob;
+using Microsoft.Azure.Storage.RetryPolicies;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ImportExportDevices
+{
+    public static class IOTHubDevices
+    {
+        //IoT Hub connection string. You can get this from the portal.
```

Comment:
NIT: missing whitespace

Reply:
after the  opening curly-brace for the class? I have no problem making the code consistent to the way everybody else does this, I just need to know what it is...

Reply:
Please follow our coding guidelines: https://github.com/Azure/azure-iot-sdk-csharp/blob/master/doc/coding-style.md